indico/modules/designer/client/js/index.js

```diff
@@ -594,6 +594,17 @@ import {$T} from 'indico/utils/i18n';
       );
       return;
     }
+    for (const item of Object.values(items)) {
+      if (item.type === 'fixed_image' && !item.image_id) {
```

Comment:
I think you can simplify this check like this:

```js
if (Object.values(items).some(item => item.type === 'fixed_image' && !item.image_id)) {
  ...
}
```

Reply:
Okay, thanks